### PR TITLE
Changes requested by @malinthaprasan

### PR DESCRIPTION
**Pull request by thisaltennakoon**

### CODE REVIEW

**Review by tmkasun — APPROVED** [review]

**Comment by tmkasun** [comment]
@thisaltennakoon Can you please resolve the merge conflicts

**Comment by thisaltennakoon** [comment]
@tmkasun Conflicts have been resolved

**Comment by malinthaprasan** [comment]
There are some tests failing. Are those addressed by https://github.com/wso2/product-apim/pull/10370?
If so can we merge both PRs?

**Review by malinthaprasan — APPROVED** [review]

**Comment by thisaltennakoon** [comment]
@malinthaprasan Yes. Test failure will get resolved after merging these two PRs

File: components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/Pagination.java [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/*
+ *  Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (npamudika) [inline components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/Pagination.java:2]: Let's modify the year in the license.

Comment (thisaltennakoon) [reply components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/Pagination.java:2]: Fix in https://github.com/wso2/carbon-apimgt/pull/11907

**Comment by shilmyhasan** [comment]
Lets add a meaningfull title and description to the PR.

**Pull request merged**